addons/reload/initKeybinds.sqf

```diff
@@ -0,0 +1,16 @@
+#include "\a3\ui_f\hpp\defineDIKCodes.inc"
+
+// Add Keybinds
+["ACE3 Weapons", QGVAR(checkAmmo), LSTRING(checkAmmo), {
+    // Conditions: canInteract
+    if !([ACE_player, vehicle ACE_player, ["isNotInside", "isNotSwimming", "isNotSitting"]] call EFUNC(common,canInteractWith)) exitWith {false};
```

Comment:
```suggestion
    if !([ACE_player, objectParent ACE_player, ["isNotInside", "isNotSwimming", "isNotSitting"]] call EFUNC(common,canInteractWith)) exitWith {false};
```
Copied from `fnc_getAmmoToLinkBelt.sqf` above. Dunno if that works as intended, but I don't really see why not.

Reply:
`vehicle` is more appropriate in this case.

Reply:
How come ?

Reply:
canInteractWith conditions could potentially rely on `_unit == _target` (if unit is on foot, `vehicle _unit` returns `_unit`

Reply:
Nothing can be simple in SQF ...